tests/test_stepstoreproduce.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from bugbug.models.stepstoreproduce import StepsToReproduceModel
+
+
+def test_get_labels():
+    model = StepsToReproduceModel()
+    classes, _ = model.get_labels()
+    assert classes[1488310]
+    assert not classes[1372243]
```

Comment:
Could you also add a bug which is not in classes (you can use one of the bugs which is already in the mock DB and has `cf_has_str` not set to `yes` and not set to `no`).

Reply:
assert XXX not in classes

Reply:
You mean choose a bug that has `cf_has_str : ---- ` and assert that it is not in classes?

Reply:
Yes!

Reply:
Done

Reply:
Done kindly review.This should suffice IMO :)